app/api/ada/lib/lovefieldDatabase.js

```diff
@@ -57,40 +86,55 @@ export const loadLovefieldDB = () => {
     .addColumn(txAddressesTableSchema.properties.address, Type.STRING)
     .addColumn(txAddressesTableSchema.properties.tx, Type.STRING)
     .addPrimaryKey([txAddressesTableSchema.properties.id])
+    // Address must also be part of the AddressesTable
     .addForeignKey('fkAddress', {
       local: txAddressesTableSchema.properties.address,
       ref: `${addressesTableSchema.name}.${addressesTableSchema.properties.id}`
     })
+    // Transaction must also be part of the TxsTable
     .addForeignKey('fkTx', {
       local: txAddressesTableSchema.properties.tx,
       ref: `${txsTableSchema.name}.${txsTableSchema.properties.id}`
     });
 
-  return schemaBuilder.connect().then(newDb => {
+  return schemaBuilder.connect(
+    // Uncomment to allow debuggin with Lovefield DB Inspector plugin
```

Comment:
what plugin did you use? is it too different from using the Chrome sql inspector?

Reply:
Oh I didn't know that the Chrome could debug lovefield DB also. I'll remove this then.